### PR TITLE
Add Criteo blog

### PR DESCRIPTION
**Pull request by itoumlilt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +101,12 @@ Curious to know how big companies are operating their kafka fleet in production?
 - [Kafka infrastructure renovation at Coinbase](https://www.coinbase.com/blog/kafka-infrastructure-renovation) - `2022` - :books:
 - [How we scaled data streaming at Coinbase using AWS MSK](https://www.coinbase.com/blog/how-we-scaled-data-streaming-at-coinbase-using-aws-msk) - `2021` - :books:
 
+## Criteo
+
+- [Upgrading Kafka on a large infra, or: when moving at scale requires careful planning](https://medium.com/criteo-engineering/upgrading-kafka-on-a-large-infra-3ee99f56e970) - `2019` - :books:
+- [How Criteo is managing one of the largest Kafka Infrastructure in Europe](https://www.slideshare.net/RicardoPaiva17/how-criteo-is-managing-one-of-the-largest-kafka-infrastructure-in-europe) - `2019` - :books:
+- [Managing Kafka and Data Streams at Criteo](https://medium.com/criteo-engineering/managing-kafka-and-data-streams-at-criteo-566ffbfda6ba) - `2023` - :books:
```
Comment (dttung2905) [inline README.md:108]: nit: can we move this to the top of the list for Criteo as the list is in descending order ? :pray:

**Review by dttung2905 — CHANGES_REQUESTED** [review]
Hi @itoumlilt , thank you very much for the PR. Just a very minor comment and we are good to go

File: README.md (hunk 1 above)
Comment (itoumlilt) [reply README.md]: Hello @dttung2905 , 
Thanks for your comment, it's done.

**Comment by dttung2905** [comment]
LGTM ! Thanks again for the contribution :rocket:

**Pull request merged**